cmd/krew/cmd/internal/setup_check_test.go

```diff
@@ -43,6 +43,40 @@ func TestIsBinDirInPATH_secondRun(t *testing.T) {
 	}
 }
 
+// Test case when PATH content is not well normalized
+func TestIsBinDirInPATH_NonNormalized(t *testing.T) {
+	tempDir := testutil.NewTempDir(t)
+
+	// backup and restore initial PATH value
+	backupPath := os.Getenv("PATH")
+	defer func(backupedPath string) {
+		err := os.Setenv("PATH", backupedPath)
+		if err != nil {
+			t.Errorf("fail to restore PATH env variable after test")
```

Comment:
```suggestion
			t.Fatalf("os.Setenv(todo) failed with %v", todo, err)
```
It is better to use Fatalf here, because we want to make sure that no other test runs in this process. Otherwise, this test could fail erroneously due to the polluted PATH variable.